src/components/MediaModal/MediaModal.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { composeStories } from '@storybook/testing-react';
+
+import * as stories from './MediaModal.VisualTests.stories';
+
+const {
+  LandscapeImage,
+  PortraitImageFooter,
+  TitleDescriptionPortraitImage,
+  PortraitImageTitleDescriptionFooter,
+} = composeStories(stories);
```

Comment:
Reusing our chromatic stories for unit testing

Reply:
This is nice! I am slightly concerned with coupling these though (E.G breaking a unit test merely by changing the docs around) but perhaps it is unwarranted. Will leave it up to you.